bats/tests/containers/run-rancher.bats

```diff
@@ -1,6 +1,7 @@
 load '../helpers/load'
 
 @test 'factory reset' {
+    RD_RAMDISK_SIZE=$((12 * 1024)) # We need more disk to run the Rancher image.
```

Comment:
> it should just be a sparse disk

I checked, and the ramdisk does indeed only use the memory occupied by files, but there is no `trim` operation, so it will not return memory for blocks no longer used by files.

But I think the argument holds that `diffdisk` is a sparse file, so only the disk size used should be allocated in the ramdisk.

There may be complications for VZ mode, as RAW images are still sparse files, but use the full allocation right from the start instead of growing incrementally like QCOW2. But we'll see this right away, as the full size of the `diffdisk` file is 100GB...

Reply:
Made it use a different variable (`RD_FILE_RAMDISK_SIZE`) for per-file overrides.  The user-provided value will win over that.

Made the size be in gigabytes.